TeacherWorkout.Api/GraphQL/Inputs/LessonSaveInput.cs

```diff
@@ -0,0 +1,13 @@
+using GraphQL.Types;
+
+namespace TeacherWorkout.Api.GraphQL.Inputs
+{
+    public class LessonSaveInput : InputObjectGraphType
```

Comment:
Proposal:

Rename this to a LessonSaveInputType
Define a LessonSaveInput
Inherit LessonSaveInputType from InputObjectGraphType<LessonSaveInput>
Define the fields using the strongly typed methods to avoid typos.

Reply:
Not sure I understand this correctly. Which `strongly typed methods` are you referring to?

Also, wouldn't it be confusing if a file was named `..InputType`? Would it be both an `Input` & a `Type`? :man_shrugging:

Reply:
Made https://github.com/alex-damian-negru/teacher-workout-backend/pull/1 to show what i was talknig about.